components/list/list-item-mixin.js

```diff
@@ -317,17 +321,20 @@ export const ListItemMixin = superclass => class extends composeMixins(
 				margin: 0;
 			}
 
-			:host(:not([selection-disabled]):not([skeleton])[selected]) [slot="outside-control-container"],
-			:host(:not([selection-disabled]):not([skeleton])[selectable][_hovering]) [slot="outside-control-container"],
+			:host([_hovering-primary-action]) [slot="outside-control-container"],
+			:host([_hovering-selection]) [slot="outside-control-container"],
+			:host([_focusing-primary-action]) [slot="outside-control-container"],
 			:host(:not([selection-disabled]):not([skeleton])[selectable][_focusing]) [slot="outside-control-container"] {
-				background-color: white;
+				background-color: white !important;
```

Comment:
`!important` used to overwrite line 336 for selected list items.

Reply:
Similarly, it's not clear to me why these styles need to be changed. Couldn't we still use `_hovering` as we are with `_focusing`?

Reply:
Ah so the checks for selection-disabled and skeleton are not needed anymore because they are being checked before adding those properties? It's still probably a good idea to avoid using `!important` if we can.

Maybe instead, we could conditionally add a class to the container to give these styles higher specificity